app/lib/surface/Lank.coffee

```diff
@@ -658,7 +658,8 @@ module.exports = Lank = class Lank extends CocoClass
       d.updatePosition()
 
   addLabel: (name, style, labelOptions={}) ->
-    @labels[name] ?= new Label sprite: @, camera: @options.camera, layer: @options.textLayer, style: style, labelOptions: labelOptions
+    layer = if labelOptions.groundLayer then @options.groundLayer else @options.textLayer
```

Comment:
shouldn't it be 
```
if labelOptions.groundLayer then labelOptions.groundLayer else @options.textLayer
```

Reply:
Nope, labelOptions.groundLayer is just a boolean and we need to send there a reference to the real layer.